build.sbt

```diff
@@ -50,9 +67,9 @@ lazy val scalaApiModels = project.in(file("api-models") / "scala")
       commitReleaseVersion,
       tagRelease,
       publishArtifacts,
-      releaseStepTask(bintrayRelease),
       setNextVersion,
       commitNextVersion,
+      releaseStepCommand("sonatypeReleaseAll"),
```

Comment:
My question is which one should we put for this command? 
releaseStepCommandAndRemaining("sonatypeRelease") OR
releaseStepCommand("sonatypeReleaseAll") OR
releaseStepCommandAndRemaining("publishSigned") OR
releaseStepCommand("sonatypeBundleRelease")